sdcm/argus_results.py

```diff
@@ -117,6 +117,24 @@ class Meta:
         }
 
 
+class BackupResult(GenericResultTable):
+    class Meta:
+        name = "Backup benchmark"
+        description = "Backup benchmark"
+        Columns = [
+            ColumnMetadata(name="backup time", unit="s", type=ResultType.DURATION, higher_is_better=False),
+        ]
+
+
+class ReadResult(GenericResultTable):
+    class Meta:
+        name = "Read timing"
+        description = "Read timing"
+        Columns = [
+            ColumnMetadata(name="read time", unit="s", type=ResultType.DURATION, higher_is_better=False),
+        ]
```

Comment:
I'd suggest to show connection to Manager in class names.
What do you think about renaming to `ManagerBackupBenchmarkResult` and `ManagerBackupBenchmarkReadResult`?
Yes, it's more complex but pretty straightforward for everyone.

Reply:
I dont mind having whatever name for the class, but why it should be connected to the Manages? Is it possible that someone would use this table to report read timing to argus not in a context of manager?

Reply:
Good question btw.
@soyacz What approach do you prefer?

Reply:
Let's add `ManagerBackup` prefix as currently it sounds too generic. 
Unless you decide to refactor and put Manager related stuff out of this file to separate one somewhere around manager code - then prefixes are redundant.

Btw. from my uninvolved perspective I don't know even what is `read` - does it refer to backup or reading from db? - descriptions could be more descriptive so anyone could understand it.

Reply:
Done